prac_05/word_occurrences.py

```diff
@@ -4,3 +4,14 @@
 Actual:    minutes
 """
 
+text = input("Enter your text: ")
+words = text.split()
```

Comment:
Maybe normal_words is a better name here or something more specific.

Reply:
Sounds good